Makefile

```diff
@@ -1,41 +1,3 @@
-TARGETS ?= darwin/amd64 linux/amd64 windows/amd64
-DIST_DIRS = find * -type d -exec
-
-# go options
-BINDIR := $(CURDIR)/bin
-GO ?= go
-GOFLAGS :=
-TAGS :=
-LDFLAGS := -w -s
-
-.PHONY: build
-build:
```

Comment:
This was for building the install executable, I have removed that since we don't need an executable to install the CRDs, people can do `kubectl apply -f ./crds`.